### PR TITLE
A thank you to all those who've contributed to PyVista's success!

### PR DESCRIPTION
**Pull request by banesullivan**
Thank you all for contributing to PyVista!! We are so grateful for contributors who have added new features, squashed bugs, fixed documentation typos, shared new examples, and so much more -- all of these efforts go a long way in making PyVista what it is. To show our appreciation, we want to put your name on the collective work, adding many of you to The PyVista Developers list in no particular order:

- @adeak
- @tkoyama010
- @rodrigomologni
- @whophil
- @thomgrand
- @larsoner

I hope it is safe to assume consent to add your name here - please let us know otherwise (either in a comment here or privately on Slack or [email](mailto:info@pyvista.org))

@pyvista/developers, if there is anyone else you feel I forgot to add here, please let me know or add them directly to this PR

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -17,3 +17,9 @@ considered "The PyVista Developers":
 - Henrik Åhl, (`@supersubscript <https://github.com/supersubscript/>`_)
 - Guillaume Favelier, (`@GuillaumeFavelier <https://github.com/GuillaumeFavelier/>`_)
 - Jevin Jones, (`@JevinJ <https://github.com/JevinJ/>`_)
+- Andras Deak, (`@adeak <https://github.com/adeak>`_)
```
Comment (adeak) [inline AUTHORS.rst:20]: Might as well use my True Name ;) 
```suggestion
- András Deák, (`@adeak <https://github.com/adeak>`_)
```

**Review by adeak — APPROVED** [review]

**Review by tkoyama010 — APPROVED** [review]
I'm honored. Thank you :-)

**Comment by rodrigomologni** [comment]
I'm honored too. Thank you!

**Comment by akaszynski** [comment]
Thanks for doing this @banesullivan!

**Review by GuillaumeFavelier — APPROVED** [review]
Excellent idea :ok_hand:

**Comment by thomgrand** [comment]
Thanks, I'm honored as well! :)
Pyvista remains my go-to library for VTK related tasks in python and I hope it will continue to thrive.

**Pull request merged**